borg/upgrader.py

```diff
@@ -267,10 +267,14 @@ def find_key_file(cls, repository):
         get_keys_dir = cls.get_keys_dir
         id = hexlify(repository.id).decode('ascii')
         keys_dir = get_keys_dir()
+        def ex():
+            raise KeyfileNotFoundError(repository.path, keys_dir)
```

Comment:
thanks for trying to avoid code duplication, but in this case i think have 1 line duplicate is preferrable to adding a function with just that one line.